### PR TITLE
chore(doc): add databend cloud

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Add  databend cloud

## Changelog

- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #issue

### CODE REVIEW

File: docs/doc/01-cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Worksheets is a powerful SQL editor where you can run SQL queries. For example, you can now do [Conversion Funnel Analysis](../90-learn/04-analyze-funnel-with-databend.md) online.
+
+<img src="/img/cloud/databend_cloud_worksheet_demo.png"/>
+
+### Connect to a Serverless Warehouse on Databend Cloud
+
+Databend Cloud provides a connection string for your applications to connect to it:
+
+```shell
+https://<tenant>--<warehouse>.ch.aws-us-east-2.default.databend.com/
```
Comment (ZhiHanZ) [inline docs/doc/01-cloud/index.md:53]: I think in here user already get their account tenant id, thus it is a fixed name rather than configurable

Comment (BohuTANG) [reply docs/doc/01-cloud/index.md:53]: What the endpoint URL should be?

**Comment by BohuTANG** [comment]
@Xuanwo 

There is a question that needs your help, in my local, `yarn start` the `Databend Cloud` shows in the tab list:
image

But from the Vercel preview: https://databend-git-fork-bohutang-doc-databend-cloud-databend.vercel.app/ the tab is not working:
image

File: docs/doc/01-cloud/index.md (hunk 1 above)
Comment (ZhiHanZ) [reply docs/doc/01-cloud/index.md:53]: like `<FQDN for any of your warehouse clickhouse http host>`?

**Comment by Xuanwo** [comment]
> There is a question that needs your help, in my local, `yarn start` the `Databend Cloud` shows in the tab list:

I'm guessing you are using the out-of-date version of docusaurus. Try `yarn install` to update them.

And I fixed the `_category_.json`, PTAL.

**Comment by Xuanwo** [comment]
image

**Pull request merged**